exercises/concept/squeaky-clean/.docs/instructions.md

```diff
@@ -29,11 +29,14 @@ SqueakyClean.clean("a-bc");
 ## 3. Convert leetspeak to normal text
 
 Modify the (_static_) `SqueakyClean.clean()` method to convert [leetspeak][leet-speak] to normal text.
+
 For simplicity we will only be using `4`, `3`, `0`, `1` and `7` from the table.
```

Comment:
The following suggestion is from the discussion in the [forum post](http://forum.exercism.org/t/java-squeaky-clean-unclear-readme/13575):

```suggestion
For simplicity we will only be replacing 4, 3, 0, 1 and 7 with a, e, o, l, and t, respectively.
```